src/components/material/infomedia/InfomediaModal.tsx

```diff
@@ -9,18 +9,18 @@ import { Manifestation } from "../../../core/utils/types/entities";
 export const infomediaModalId = (pid: Pid) => `infomedia-modal-${pid}`;
 
 interface InfomediaModalProps {
-  mainManifestation: Manifestation;
-  infoMediaId: string;
+  selectedManifestations: Manifestation[];
+  infoMediaIds: string[];
 }
 
 const InfomediaModal: React.FunctionComponent<InfomediaModalProps> = ({
-  mainManifestation: { pid },
-  infoMediaId
+  selectedManifestations,
+  infoMediaIds
 }) => {
   const t = useText();
 
   const { data, error } = useGetInfomediaQuery({
-    id: infoMediaId
+    id: infoMediaIds[0]
   });
```

Comment:
I reworked it so that we just use one infomedia ID. However, when comes to the modal ID, it works perfectly fine with post IDs, and I don't see a reason to change it. In this case I favour the modal ID usage being uniform.